.ci/docker/aotriton_version.txt

```diff
@@ -1,5 +1,5 @@
 0.7b
-manylinux_2_17
+manylinux_2_17;manylinux_2_17
 rocm6.1;rocm6.2
 9be04068c3c0857a4cfd17d7e39e71d0423ebac2
 006f4d982c9a9c768f31f0095128705fecb792136827e2456241fe79764de7a4;3e9e1959d23b93d78a08fcc5f868125dc3854dece32fd9458be9ef4467982291
```

Comment:
@xinyazhang @jeffdaily Also wondering if it makes sense to keep this file in `.ci/docker` anymore, since it's only referred to by `cmake/External/aotriton.cmake` now, and is therefore not used in the CI docker image builds at all.

Reply:
I have the same concern, but couldn't find a proper location for this txt file. `cmake` should be a good place but no other existing instances.

Reply:
If it isn't used by .ci/docker, it shouldn't be there.  Can we hard-code these into aotriton.cmake?

Reply:
@xinyazhang Yes, that would also simplify the code a little bit since it wouldn't need to read and parse the txt file?

Reply:
Yes if we don't need a text file shared by CI and cmake we can certainly put all the data into this cmake file.
(Although the SHA256 list would be too long and may need some tricks to format it)

Reply:
There's sha1sum, which is a bit smaller. As an example:
```
# sha1sum /data/tmp/wheels/pytorch2.1_rocm5.7/torchvision-0.16.0+rocm5.7-cp310-cp310-linux_x86_64.whl
c0b96451d1d7916a5cbe467e40d29b5f22572a3b  /data/tmp/wheels/pytorch2.1_rocm5.7/torchvision-0.16.0+rocm5.7-cp310-cp310-linux_x86_64.whl

# sha256sum /data/tmp/wheels/pytorch2.1_rocm5.7/torchvision-0.16.0+rocm5.7-cp310-cp310-linux_x86_64.whl
defc5e7382934394f6c664a7de223713218674527f99e66becd31bd387eb66f0  /data/tmp/wheels/pytorch2.1_rocm5.7/torchvision-0.16.0+rocm5.7-cp310-cp310-linux_x86_64.whl
```

Reply:
Let's address the removal of the txt file in the original PR.

Reply:
SHA1 is considered as insecure and AFAIK 256-bits is the minimal requirement to ensure security for all known cryptographic hash algorithms